src/makeClientFunctionWrapper.js

```diff
@@ -18,16 +18,25 @@ function makeClientFunctionWrapper({
 }) {
   return async function(browserFunction, dependencies = {}) {
     const getResultSize = clientFunctionExecuter(
-      () =>
-        browserFunction().then((result = {}) => {
+      () => {
+        // eslint-disable-next-line no-undef
+        return browserFunction(functionArgs).then((result = {}) => {
```

Comment:
why isn't this `dependencies.functionArgs`? How does this work if it's undefined?

Reply:
`dependencies` doesn't exist in this context, just its destructured contents.

Reply:
I must be missing something. `dependencies` is the second parameter to the wrapping function in line 19. Is there some magic going on related to TestCafe?